staff.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Staff
+---
+
+## Identifying Staff
+
+The _staff_ are the volunteers who act as network and system administrators to
+keep Libera.Chat running smoothly.
+
+The complete current list of staff is available using the IRC command
+`/who libera/staff/*`.
+
+When this page was last updated, the staff members were:
+
+staff|staff|staff|staff
+--|--|--|--
+amdj     | emerson   | JonathanD  | nhandler
+bigpresh | emilsp    | jose       | nicole
+D[\_]     | Freso     | kline      | niko
+deedra   | Fuchs     | lausefuchs | njan
+edk      | glguy     | li         | Stx
+el       | jess      | mniip      | Swant
+
+Among the ops in `#libera` is also a service bot (which is not currently a
+member of staff):
+
+| bot
+|-
+| ozone
+
+## Contacting Staff
+
+### On IRC
+
+Assuming you can connect to IRC (`ircs://irc.libera.chat:6697`), that should be
+your first point of contact with staff.
+
+The quickest way to find the list of on-line staff is to look for chanops among
```

Comment:
Given the update to use `/who` I don't think this should be included.

Reply:
Chanops are not necessarily staff and staff are not necessarily Chanops. This section is not correct. `/stats p` and if empty `/who libera/staff/*` are the correct ways

Reply:
After 11 years on IRC and I'm still learning.

My IRC client didn't have `/stats` until I added it recently, and still doesn't have `/who` or `/query`.

To be honest, the output from `/who` is dense and inscrutable; to the novice it's not at all clear which field is the nick that one is supposed to target. (Even as I was using it to write the staff list in this PR, I found myself getting muddled as to which was the account name and which was the current nick.)

Compare that with the `#libera` chanops list which shows the actual nicks that are in use, is always visible in many clients, and on the whole is a _reasonable_ approximation to the list of available staff (give or take some bots). Even if it's not perfect, it's simpler for users.

So I'm inclined to document all three methods, with the caveat that not all chanops are staff.

My understanding is that the general advice is to try these in order until you get a response:
1. If it's not private, announce your request in `#libera`;
2. Use `/stats p` to find an on-duty staffer, and PM them;
3. Use `/who libera/staff/*` to find a logged-in staffer, and PM them;
4. PM one of the chanops in `#libera`, but avoid bots such as `ozone`;
5. Read the instructions on how to configure your client to use `/stats` or `/who` if your client does not have those built in, and the instructions on how to interpret the output of `/who`.
6. Wait 15 minutes and start again from the top.

There are probably differing opinions on whether using the channel should be first, but it comes somewhere in the first 3, before pinging chanops.

Reply:
Clients are already responsible for parsing output of server numerics (for example, in reply to `/WHOIS`). Parsing `/WHO` output is not difficult.